base/sysinfo.jl

```diff
@@ -98,7 +98,19 @@ Standard word size on the current machine, in bits.
 """
 const WORD_SIZE = Core.sizeof(Int) * 8
 
-global SC_CLK_TCK::Clong, CPU_NAME::String, JIT::String
+global SC_CLK_TCK::Clong
+"""
+    Sys.CPU_NAME::String
+
+A string representing the name of CPU.
+"""
+global CPU_NAME::String
+"""
+    Sys.JIT::String
+
+A string representing the specific Just-In-Time (JIT) compiler being utilized in the current runtime.
```

Comment:
Would be good to give an example or two.

e.g. on my machine with Julia 1.10 it is `"ORCJIT"` for the LLVM "ORC" ("On-Request Compilation") JIT library.